hbase-protocol-shaded/src/main/protobuf/server/master/Master.proto

```diff
@@ -1123,6 +1132,9 @@ service MasterService {
 
   rpc UpdateRSGroupConfig(UpdateRSGroupConfigRequest)
   returns (UpdateRSGroupConfigResponse);
+
+  rpc GetBalancerDecisions(BalancerDecisionsRequest)
```

Comment:
Remove. Use getLogEntries() for this instead.

Reply:
Same as above:
Using common API for proto seems almost impossible. One use-case of slowLog is being handled by RSRpc services whereas the other for balancerDecision is being managed by MasterRpc services. Both have different sets of Rpc implementations written and handled at different places.
Moreover, at RPC level, we can't achieve abstraction since protobuf doesn't support `is-a` relationship among proto `messages`.